lib/travis/build/script/go.rb

```diff
@@ -103,6 +108,13 @@ def uses_make
             '-f GNUmakefile || -f makefile || -f Makefile || -f BSDmakefile'
           end
 
+	  def uses_15_vendoring
+            if (go_version < '1.5' || go_version == 'go1')
+              'false'
+            end
```

Comment:
Which notably does what you might not expect when `go_version == 'go1'` as you've called out in the `if` above.